skyportal/plot.py

```diff
@@ -261,6 +277,22 @@ def photometry_plot(obj_id, user, width=600, height=300, device="browser"):
 
     data['color'] = [get_color(f) for f in data['filter']]
 
+    # get marker for each unique instrument
+    instruments = list(data.instrument.unique())
+    markers = []
+    for i, inst in enumerate(instruments):
+        if i >= len(phot_markers):
+            pmi = i - len(phot_markers)
+        else:
+            pmi = i
+        markers.append(phot_markers[pmi])
```

Comment:
```suggestion
        markers.append(phot_markers[i % len(phot_markers)])
```
If there are more than `2 * len(phot_markers)` than the current logic will fail. Using the mod will guard against that (although it's probably very unlikely that we ever get that many instruments on a source)